framework/test/unit/modules/interoperability/mainchain/commands/terminate_sidechain_for_liveness.spec.ts

```diff
@@ -68,7 +68,7 @@ describe('TerminateSidechainForLivenessCommand', () => {
 				lastCertificate: {
 					height: 10,
 					stateRoot: utils.getRandomBytes(32),
-					timestamp: Date.now(),
+					timestamp: Math.floor(Date.now() / 1000),
```

Comment:
this is not related to this issue?

Reply:
timestamp here should be without milliseconds otherwise it will exceed 2**64-1.

Also this timestamp compares with header timestamp which also uses w/o ms format